static/app/components/replays/header/replayMetaData.tsx

```diff
@@ -24,13 +24,13 @@ function ReplayMetaData({replayErrors, replayRecord}: Props) {
   const referrer = getRouteStringFromRoutes(routes);
   const eventView = EventView.fromLocation(location);
 
-  const domEventsTab = {
+  const breadcrumbTab = {
     ...location,
     query: {
       referrer,
       ...eventView.generateQueryStringObject(),
-      t_main: 'dom',
-      f_d_type: 'ui.slowClickDetected',
+      t_main: 'breadcrumbs',
```

Comment:
nit, but i think we can import `TabKey` here and use it. helpful for finding the places again later.